content/en/docs/instrumentation/php/manual.md

```diff
@@ -78,11 +78,11 @@ To do [Tracing](/docs/concepts/signals/traces/) you'll need to acquire a
 **Note:** Methods of the OpenTelemetry SDK should never be called.
 
 First, a `Tracer` must be acquired, which is responsible for creating spans and
-interacting with the [Context](#context-propagation). A tracer is acquired by
-using the OpenTelemetry API specifying the name and version of the [library
-instrumenting][instrumentation library] the [instrumented library] or
-application to be monitored. More information is available in the specification
-chapter [Obtaining a Tracer].
+interacting with the [Context](/docs/instrumentation/php/propagation). A tracer
```

Comment:
Sorry for the churn in suggestions, but let's use a relative path here and elsewhere in the document for PHP-specific pages. And use the path normal form of ending with a slash. Thanks!

```suggestion
interacting with the [Context](../propagation/). A tracer
```